docs/integrations/destinations/aws-datalake.md

```diff
@@ -1,59 +1,40 @@
 # AWS Datalake
 
-## Overview
+This page contains the setup guide and reference information for the AWS Datalake destination connector.
 
 The AWS Datalake destination connector allows you to sync data to AWS. It will write data as JSON files in S3 and 
-update the Glue data catalog so that the data is available throughout other AWS services such as Athena, Glue jobs, EMR, 
-Redshift, etc.
+will make it available through a [Lake Formation Governed Table](https://docs.aws.amazon.com/lake-formation/latest/dg/governed-tables.html) in the Glue Data Catalog so that the data is available throughout other AWS services such as Athena, Glue jobs, EMR, Redshift, etc.
 
-### Sync overview
-#### Output schema
-
-The Glue tables will be created with schema information provided by the source, i.e : You will find the same columns
-and types in the destination table as in the source.
-
-#### Features
-
-| Feature | Supported?\(Yes/No\) | Notes |
-| :--- | :--- | :--- |
-| Full Refresh Sync | Yes |  |
-| Incremental - Append Sync | Yes |  |
-| Namespaces | No |  |
-
-## Getting started
-### Requirements
+## Prerequisites
 
 To use this destination connector, you will need:
-* A AWS account
-* A S3 bucket where the data will be written
-* A AWS Lake Formation database where tables will be created (one per stream)
+* An AWS account
+* An S3 bucket where the data will be written
+* An AWS Lake Formation database where tables will be created (one per stream)
 * AWS credentials in the form of either the pair Access key ID / Secret key ID or a role with the following permissions:
 
     * Writing objects in the S3 bucket
     * Updating of the Lake Formation database
 
-See the setup guide for more information about the creation of the resources.
-
-### Setup guide
-#### Creating an AWS account
+### Creating an AWS account
```

Comment:
With these sub-headings here for creating different accounts. Especially, since you have links to external docs - let's move them to the Setup Guide section as part of the individual steps.

You can see an example of this here: https://docs.airbyte.com/integrations/sources/s3/#step-2-set-up-the-amazon-s3-connector-in-airbyte. So instead of saying "creating an S3 bucket" as a separate heading....it would be in the steps of the Setup Guide like "2. Enter your [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) credentials by using the [the AWS policy generator](https://awspolicygen.s3.amazonaws.com/policygen.html)" Or something similar.

Reply:
This means the Prerequisites are short and succinct and easy to reference for a user.